include/disk.h

```diff
@@ -0,0 +1,13 @@
+#ifndef _DISK_H
+#define _DISK_H
+
+#include "types.h"
+
+// based on how other "gClass" works, we can assume this was the struct used for Disk
```

Comment:
```suggestion
// __anon_0x73B29
```
turns out this one is in the debug info after all